### ISSUE TITLE
'time' key and value from the event hash results in windows that cannot be manipulated under KDE5

### ISSUE THREAD

**Issue by bdperkin** [duncs/clusterssh#46]
I've created a patch that seems to work to resolved the, quite common among KDE5 users, problem of using cssh and then having unmoveable windows.  Although I'm not convinced the issue is with cssh, this workaround seems to fix the glitch until the KDE side can fix it.  It also, doesn't seem to, cause any problems.  I cannot seem to attach in this bug report, however, it is a small patch, so I'll add it inline here:

```patch
From 395f9ab3f6e9b97ffca6403cdf693582ddd2079e Mon Sep 17 00:00:00 2001
From: Brandon Perkins <bperkins@redhat.com>
Date: Fri, 6 Nov 2015 16:19:03 -0500
Subject: [PATCH] Remove 'time' key from the event hash.

---
 lib/App/ClusterSSH.pm | 2 --
 1 file changed, 2 deletions(-)

diff --git a/lib/App/ClusterSSH.pm b/lib/App/ClusterSSH.pm
index 50bb8e8..179d3b8 100644
--- a/lib/App/ClusterSSH.pm
+++ b/lib/App/ClusterSSH.pm
@@ -572,7 +572,6 @@ sub send_text($@) {
                     'name'        => $event,
                     'detail'      => $code,
                     'state'       => $state,
-                    'time'        => time(),
                     'event'       => $servers{$svr}{wid},
                     'root'        => $xdisplay->root(),
                     'same_screen' => 1,
@@ -1689,7 +1688,6 @@ sub key_event {
                     'name'        => $event,
                     'detail'      => $keycode,
                     'state'       => $state,
-                    'time'        => time(),
                     'event'       => $servers{$_}{wid},
                     'root'        => $xdisplay->root(),
                     'same_screen' => 1,
-- 
2.5.0


```

There are a number of related bugs around, I'll list a few here:

https://bugs.kde.org/show_bug.cgi?id=348569
https://bugzilla.redhat.com/show_bug.cgi?id=1254254
https://bugs.kde.org/show_bug.cgi?id=351490
https://bugreports.qt.io/browse/QTBUG-28398
https://bugs.launchpad.net/ubuntu/+source/qt4-x11/+bug/479740
https://bugs.gentoo.org/show_bug.cgi?id=276527
http://comments.gmane.org/gmane.network.vnc.tigervnc.user/631
https://bugzilla.redhat.com/show_bug.cgi?id=1235024
https://bugzilla.redhat.com/show_bug.cgi?id=1253496

**Comment by duncs** [comment]
Thanks for the patch.  Applied and will be released in 4.04_01

**Comment by luebking** [comment]
fyi, time() returns seconds since epoch, but given the patch you require the X server timestamp (typically miliseconds since last server start) - no idea on how to access that in perl, though.
